resources/main.css

```diff
@@ -496,6 +496,40 @@ section#instructions .section-content > * {
     flex-direction: column;
 }
 
+section#details h1 {
+    margin-bottom: 10px;
+}
+
+section#details .non-standard-params {
+    display: none;
+    text-align: center;
+    margin-bottom: 2em;
+}
+
+section#details .non-standard-params h2 {
+    color: var(--highlight);
+}
+
+#non-standard-params-table {
+    border-collapse: collapse;
+    text-align: left;
```

Comment:
I'd use `center`, at least for the 2 columns `default` and `value`

and/or add some padding to the `td`

![image](https://github.com/user-attachments/assets/b0dbaee9-1e00-43dd-80ca-1f920c2b6540)

Reply:
I had  `th` in `#non-standard-params-table tbody th` instead of `td`.
Added now consistend spaceing for both the header and the body.